test/operator/installation.go

```diff
@@ -25,6 +30,52 @@ func InstallFromSource(withCsi bool) features.Func {
 	return manifests.InstallFromFiles(paths)
 }
 
+func InstallViaMake(withCSI bool) features.Func {
+	return func(ctx context.Context, t *testing.T, environmentConfig *envconf.Config) context.Context {
+		rootDir := project.RootDir()
+
+		runBuildAndManifests(rootDir, t)
+
+		platform := kubeobjects.ResolvePlatformFromEnv()
+		makeTarget := "deploy"
+		switch platform {
+		case kubeobjects.Openshift:
+			makeTarget = strings.Join([]string{makeTarget, "openshift"}, "/")
+		case kubeobjects.Kubernetes:
+			makeTarget = strings.Join([]string{makeTarget, "kubernetes"}, "/")
+		default:
+			t.Fatal("failed to install the operator via the make command as no correct platform was set")
+			return nil
+		}
+
+		if !withCSI {
+			makeTarget = strings.Join([]string{makeTarget, "no-csi"}, "-")
+		}
```

Comment:
nitpick: maybe a bit overkill but you could move this into a helper function AND unittest it, as its just simple string concats with easy inputs and outputs to check for

Reply:
something like:
```
getDeployMakeTarget(platform string, withCSI bool) string {...}
```